docs/licensing.md

```diff
@@ -0,0 +1,146 @@
+
+ \* **Ixia-c Traffic Port Capacity** is determined as a sum of configured Ixia-c test port speeds with possible values: 100GE, 50GE, 40GE, 25GE, 10GE, 1GE. Maximum data plane performance of an Ixia-c port may be less than the configured port speed, depending on capabilities of the underlying hardware and software drivers.
+
+ \*\* **Test seat concurrency** applies to a number of controller instances running with a configuration that exceeds the capabilities of the Community Edition.
+
+ \*\*\* **Restricted** protocol scale supports maximum 4 BGP sessions per test. Capabilities of the **Limited** protocol scale depend on the protocol. Please contact [Keysight Support](https://support.ixiacom.com/contact/support) for details.
+
+ Keysight Elastic Network Generator can simultaneously consume multiple licenses to increase the capabilities of a test. For example, if the Ixia-c Traffic Port Capacity configured in one test is 100GE, two Developer licenses will be consumed if available.
+
+ If you require capabilities beyond those provided by the Community Edition, use [Keysight Elastic Network Generator](https://www.keysight.com/us/en/products/network-test/protocol-load-test/keysight-elastic-network-generator.html) product page to request an evaluation or a quote.
+
+## License Server
+
+Keysight uses a license server to manage floating or network shared licenses for its software products. The license server enables licenses to float and not be tied to a specific Elastic Network Generator instance. The Elastic Network Generator controllers MUST be able to reach the License server.
+
+## Requirements
+
+### Network connectivity
+
+1. Internet access from the Keysight License server VM over HTTPS to the Keysight Software Manager (KSM) service [https://ksm.software.keysight.com/](https://ksm.software.keysight.com/) is desirable for online license activation, but not required. Alternative offline activation method is available and only requires a license administrator to have access to KSM.
+2. Access from a license administrator's computer over SSH (TCP/22) for license operations (activation, deactivation, reservation, sync).
+3. Access from any Keysight Elastic Network Generator `ixia-c-controller` container that needs a license during a test run over gRPC (TCP/7443) for license checkout and check-in.
+
+The diagram below illustrates how different components communicate with the License server:
+
+![License Server Connectivity](./res/license-server.drawio.svg)
+
+### System requirements – KVM
+
+The License server VM resource requirements when running on KVM hypervisor:
+
+* 2 CPU cores
+* 4 GB of RAM
+* Minimum 10GB of storage
+* 1 available physical bridged adapter on KVM for management connectivity
+
+## Deployment
+
+### KVM
+
+1. Download the following license server VM deployment script:
+
+ 	 `license_server_kvm.sh`
```

Comment:
Please add a link to the license server, where this script is available for download.